spec/granite_orm/associations/belongs_to_custom_type_spec.cr

```diff
@@ -0,0 +1,54 @@
+require "../../spec_helper"
+
+class Tool < Granite::ORM::Base
```

Comment:
@drujensen now that the excellent test macro cleanup has been merged, what do you think about tweaking this a little to run on all adapters?